doc/getting-started.md

```diff
@@ -82,16 +79,16 @@ One can use the a convenient tool `dpdata` to convert data directly from the out
 A model has two parts, a descriptor that maps atomic configuration to a set of symmetry invariant features, and a fitting net that takes descriptor as input and predicts the atomic contribution to the target physical property.
 
 DeePMD-kit implements the following descriptors:
-1. [`se_e2_a`](train-se-e2-a.md#descriptor): DeepPot-SE constructed from all information (both angular and radial) of atomic configurations. The embedding takes the distance between atoms as input.
-2. [`se_e2_r`](train-se-e2-r.md): DeepPot-SE constructed from radial information of atomic configurations. The embedding takes the distance between atoms as input.
-3. [`se_e3`](train-se-e3.md): DeepPot-SE constructed from all information (both angular and radial) of atomic configurations. The embedding takes angles between two neighboring atoms as input.
+1. <a rel="subsection" href="train-se-e2-a.html#descriptor">`se_e2_a`</a>: DeepPot-SE constructed from all information (both angular and radial) of atomic configurations. The embedding takes the distance between atoms as input.
```

Comment:
It breaks the link in markdown.

Reply:
Yes. The html links are supposed to be built from markdown automatically by "sphinx-build". But only head of an external file can be automatically linked. Sphinx-build cannot create the  `rel="subsection"` text. 
So, I can either change these links to html style and abandon markdown links, or I'll have to modify the sphinx source code, and upload a new compiler to readthedocs.org to make everything work.

Reply:
It's working now. Please take a look.

Reply:
Great!